src/CloudFlare/IpRewrite.php

```diff
@@ -69,38 +82,32 @@ public static function reset()
     protected static function rewrite()
     {
         // only should be run once per page load
-        if (self::$is_cf !== NULL) {
+        if (self::$is_loaded) {
```

Comment:
even if rewrite cannot happen, `is_loaded` is true. So that check won't work. It'll have to be `is_cf` or `is_loaded`. I choose the explicit one.